exporter/coralogixexporter/package_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package coralogixexporter
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+// The IgnoreTopFunction call prevents catching the leak generated by opencensus
+// defaultWorker.Start which at this time is part of the package's init call.
+// See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m, goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"))
```

Comment:
Is the ignore here needed? I removed a bunch of these after the change to remove much of the opencensus dependencies was merged in core.

Reply:
Thanks for catching this @codeboten, you're right, this is no longer needed. I've removed it.